testsuite/run_sets/finishing.yml

```diff
@@ -14,5 +14,6 @@
 # these features are needed for gathering log/debug info
 - features/finishing/srv_debug.feature
 - features/finishing/allcli_debug.feature
+- features/finishing/minssh_sp_migration.feature
```

Comment:
Move it 3 lines above, after srv_smbda

also change "this feature is destructive" comment to "these features are destructive"